client/my-sites/media/main.jsx

```diff
@@ -119,6 +119,11 @@ export default React.createClass( {
 	},
 
 	getModalButtons() {
+		// do not render buttons if the media image editor is opened
+		if ( this.state.editedItem !== null ) {
+			return;
```

Comment:
Since this function returns an array, let's explicitly return a falsey `null` here instead of undefined.